app/classifier/tasks/survey/editor.cjsx

```diff
@@ -14,97 +14,184 @@ Papa = require 'papaparse'
 module.exports = React.createClass
   displayName: 'SurveyTaskEditor'
 
-  getInitialState: ->
-    importErrors: []
-    resettingMedia: false
-
   getDefaultProps: ->
     workflow: null
-    task: null
+    task: {}
     taskPrefix: ''
 
+  getInitialState: ->
+    importErrors: []
+    resettingFiles: false
+    editing: 'data'
+    task: JSON.parse JSON.stringify @props.task
```

Comment:
this threw me until i realized you're doing this in order to get a deep copy of the props.task object. might want to leave a comment so that nobody comes along and optimizes this out.